CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 title: Changelog
 menuTitle: Changelog
 category: Guides
-position: 103
+position: 104
```

Comment:
why do you modify the position to 104 if no files is linked to 103? Did you planned to have your getting started at 103 and not 105, which is also the position of the faq?

Reply:
@Blokje5 (I'm pinging just in case you didn't see this msg)

Reply:
Yep, that was my intent indeed. I wanted the getting started guide on position 103.